webrender/src/prim_store.rs

```diff
@@ -1068,15 +1068,14 @@ impl PrimitiveStore {
                                                       font_size_dp,
                                                       text.color,
                                                       src_glyphs[0].index,
-                                                      src_glyphs[0].x,
-                                                      src_glyphs[0].y,
+                                                      src_glyphs[0].point,
                                                       text.render_mode);
                     let mut local_rect = LayerRect::zero();
                     let mut actual_glyph_count = 0;
 
                     for src in src_glyphs {
                         glyph_key.index = src.index;
-                        glyph_key.set_subpixel_offset(src.x, src.y, text.render_mode);
+                        glyph_key.set_subpixel_offset(src.point.x, src.point.y, text.render_mode);
```

Comment:
Yes. From the first comment, I was going to do that in a follow up to make each patch smaller.